### DISCUSSION TITLE
FIXED ALLL SERVER OR GITHUB NEW VERSION ALL

### DISCUSSION THREAD

**Discussion by ford220102** [rustdesk/rustdesk#6592] [General]
Hello, I would like to ask you to repair all the github and files because I see a terrible virus, they can also burden the Internet and attacks, it is better to connect DNS support and generation, automatic cloudflare connections
FIXED ALL  conact kaspersky LAB

**Comment by ford220102** [comment]
![image](https://github.com/rustdesk/rustdesk/assets/138443348/f6ecb4b4-c902-435e-80e0-293c4d5467d3)

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/discussions/6467

**Reply by ford220102** [reply]
SCAN ALLLL VIRUS FIXED ALLL DELTE PROBLEM
please scan alll github or conact speed virus scan delte problem virustotal.com
plus please quickly scan each file and update speed Certification needs to be corrected because apparently the problem is not registered and it should also be SSL complete for security omg and remove it completely with a virus on clean mats
ALLLLLLLLLLLLLL OS SYSTEM fixed

**Comment by ford220102** [comment]
@rustdesk https://www.virustotal.com/gui/file/edd5d253206c3d925f7816328ac7dc24c89d020dae4e9d2ac2988961ce9094ff FIXED ALLL antvirus fixed alll you fixed alll build new version please

**Comment by ford220102** [comment]
@rustdesk Avast
Win64:MalwareX-gen [Trj]
AVG
Win64:MalwareX-gen [Trj]
Bkav Pro
W64.AIDetectMalware
Trapmine
Suspicious.low.ml.score

VIRUS please fixed or removed and fixed not ban help you problem connect

**Reply by ford220102** [reply]
https://www.virustotal.com/graph/embed/g345f066b75664195be33566dbe23555ff12ec09f1bb14564ae1287ee12b8ef02?theme=light

**Comment by djoudi** [comment]
![image](https://github.com/rustdesk/rustdesk/assets/152725/55a44626-5edf-46b0-956b-d860c2794ab1)

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/discussions/2485
